pulp_ansible/tests/functional/api/collection/v2/test_upload.py

```diff
@@ -15,6 +15,7 @@
 from pulp_ansible.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
 
 
+# TODO: Check if this url will stay around
```

Comment:
At least we have some control around this one.
We could base all of our tests on squeezer...

Reply:
Potentially, there are some tests that are designed for oddball collections that squeezer wouldn't be a good fit for.